app/common/patient/directives/patientSummary.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-angular.module('bahmni.common.patient').directive('patientSummary', function() {
+angular.module('common.patient').directive('patientSummary', function() {
```

Comment:
This change and the similar changes below were required to have consistent module naming. I was getting module loading errors from Angular when Karma was trying to load all the application files into the browser for testing.

If you think this is going to be a problem, let me know and we can revert.